tests/testdata/programs/ignore-scaling-changes/index.ts

```diff
@@ -61,5 +61,4 @@ const ngV2 = new eks.NodeGroupV2("ignore-scaling-config-ngv2", {
   operatingSystem: eks.OperatingSystem.AL2023,
   instanceProfile: new aws.iam.InstanceProfile("ignore-scaling-config-ngv2", {role: ngV2Role}),
   ignoreScalingChanges: true,
-  subnetIds: eksVpc.privateSubnetIds,
```

Comment:
It’d be great if we could change this to `nodeSubnetIds` instead. That way those nodes will not consume EIPs

Reply:
I can update this PR. Wasn't sure what the best fallback option here was when updating the test.